### DISCUSSION TITLE
Can't connect my website to fed.brid.gy

### DISCUSSION THREAD

**Discussion by GrassBlock1** [snarfed/bridgy-fed#1601] [Q&A] [answered]
As title shown, I tried to add my site (lab.gb0.dev) via https://fed.brid.gy/web-site but I can't.

It always shows me that it is not a [valid or supported web site](https://fed.brid.gy/docs#web-get-started). I have add `feed.rss` as rss and also a webmention in my `<head>`.

I checked the code in [web.py](https://github.com/snarfed/bridgy-fed/blob/5a19720f8f6577a1f2d5cb1f33485a45cfd42b36/web.py#L696) and I was sure that my website TLD is ok, not blacklisted.

I have tried a few times tweaking the rss but still can't get bridgy fed working, so I opened a discussion to seek help.

**Comment by GrassBlock1** [comment]
seems like a cache issue

**Reply by snarfed** [reply, marked as answer]
Sorry for the trouble! This actually is a real bug on our side, your site only started working because I poked at it. I'll follow up here with the bug fix.

**Reply by snarfed** [reply]
Fixed with 65d4f78ae2405e9a0aab893624164c0f1fda3701

**Reply by GrassBlock1** [reply]
> Sorry for the trouble! This actually is a real bug on our side, your site only started working because I poked at it. I'll follow up here with the bug fix.

Thank you for the quick fix! 

I thought it was an issue on my side so I change the site several times. I'm glad to see it works normally again now.
